commercetools-models/src/main/java/io/sphere/sdk/products/ProductDataProductDraftBuilderBase.java

```diff
@@ -72,6 +73,20 @@ public T categories(final List<Reference<Category>> categories) {
         return categories(new LinkedHashSet<>(categories));
     }
 
+    /**
+     * Adds categories to this product draft. Alias for {@link #categories(List)} which takes the o
+     *
+     * @param categories
+     * @return
+     */
+    public T categoriesObjects(final List<Category> categories) {
```

Comment:
About the naming of this method, I'm not sure if it's like this for some API or method or whatever reason. But if not, in English it should be `categoryObjects` (e.g. it's "milk boxes", not "milks boxes").